src/slack-model.jl

```diff
@@ -446,6 +446,47 @@ function NLPModels.jprod_nln!(
   return jv
 end
 
+function NLPModels.jtprod!(
+  nlp::SlackModels,
+  x::AbstractVector{T},
+  v::AbstractVector,
+  jtv::AbstractVector,
+) where {T}
+  @lencheck nlp.meta.nvar x jtv
+  @lencheck nlp.meta.ncon v
+  n = nlp.model.meta.nvar
+  @views begin
+    jtprod!(nlp.model, x[1:n], v, jtv[1:n])
```

Comment:
is @views begin really necessary? I feel like you only need it for L459?

Reply:
Nice catch, thanks! I updated it.